homeassistant/components/zha/climate.py

```diff
@@ -560,6 +561,61 @@ async def async_preset_handler_away(self, is_away: bool = False) -> bool:
         return res
 
 
+@STRICT_MATCH(
+    channel_names={CHANNEL_THERMOSTAT}, manufacturers="Eurotronic",
+)
+class EurotronicSpiritZigbeeThermostat(Thermostat):
+    """Eurotronic Spirit Zigbee Thermostat."""
+
+    manufacturer = 0x1037
+    HOST_FLAGS_ATTR = 0x4008
+    BOOST_FLAG = 0b00000100
+    OFF_MODE_FLAG = (
+        0b00010000  # if this flag is set, off mode enabled. set also disables.
+    )
```

Comment:
Wondering if it is better to have this as `enum.BitFlag` class, makes more readable e.g. `if HostFlags.Off_Mode in raw_host_flags:`

Reply:
Would be nice, but they messed up the flags and the off mode is set via a different bit than reported and used to turn the off mode off. So BitFlags likely won't work

Reply:
right. Maybe just define two bits? one for `Off_Mode_Set`  and 2nd one for `Off_Mode_Reported` ?